src/plugins/sockets/inGameEvents.js

```diff
@@ -30,6 +30,10 @@ export async function handleInGameEvents(evData) {
       gameStore.updateReady(evData.pNum);
       return;
     }
+    case SocketEvent.SET_IS_RANKED: {
+      gameStore.updateMode(evData.isRanked);
```

Comment:
Since this func is just two lines we can put it here directly and remove it as an action. Additionally, let's rename the store variable for the alert
```suggestion
      gameStore.isRanked = evData.isRanked;
      gameStore.showIsRankedChangedAlert = true;
```